CMakeLists.txt

```
@@ -199,6 +202,14 @@ add_subdirectory(src/pybind)
 add_subdirectory(src/solver)
 
 # =============================================================================
-# Install unit database, examples and utility scripts from share
+# Install unit database from share
 # =============================================================================
-install(DIRECTORY share/ DESTINATION ${NMODL_INSTALL_DIR_SUFFIX}share)
+if (NMODL_ENABLE_LEGACY_UNITS)
+  set(LegacyY "")
+  set(LegacyN "/")
+else()
+  set(LegacyY "/")
+  set(LegacyN "")
+endif()
+configure_file(share/nrnunits.lib.in ${CMAKE_BINARY_DIR}/share/nrnunits.lib @ONLY)
+install(FILES share/nrnunits.lib DESTINATION ${NMODL_INSTALL_DIR_SUFFIX}share/nrnunits.lib)
```

```suggestion
install(FILES ${CMAKE_BINARY_DIR}/share/nrnunits.lib DESTINATION ${NMODL_INSTALL_DIR_SUFFIX}share/nrnunits.lib)
```
Maybe this is needed here?

---

No, My problem was the line above, I have to install it in `CMAKE_CURRENT_BINARY_DIR`.